composer.json

```diff
@@ -131,7 +128,7 @@
         "drupal/search_api_location": "^1.0@alpha",
         "drupal/search_api_page": "^1.0",
         "drupal/search_api_pantheon": "^8.0",
-        "drupal/search_api_solr": "^4.2",
+        "drupal/search_api_solr": "4.x-dev",
```

Comment:
You should probably pin this to a specific commit in the dev branch, otherwise every time composer install is run, we could potentially get a different version that might break something.